dcbr-api/email_service/tasks.py

```diff
@@ -56,6 +56,16 @@ def send_reminder_email():
 
 @background(queue="member-registration")
 def send_registration_email(context: dict):
+    """Send an email with the registration details to an operator. The data to be used to render the message
+       and the certificate should be passed as a dictionary with the following structure:
+       {
+           "operator": operator, # an operator object
+           "registration_number": "DCR-12345" # a string representing the registration number
+       }
```

Comment:
@anissa-agahchen: this is preliminary and you may want to change it (as well as the template mappings in the email and PDF) based on the actual data structure you are planning on passing in.